src/Augurk/Startup.cs

```diff
@@ -34,13 +36,40 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            // Add core MVC services and setup the versioned API explorer
+            services.AddMvcCore()
+                    .SetCompatibilityVersion(CompatibilityVersion.Version_2_1)
+                    .AddVersionedApiExplorer(options =>
+                    {
+                        options.GroupNameFormat = "'v'VVV";
+                        options.SubstituteApiVersionInUrl = true;
+                    });
+            
+            // Add the rest of the MVC stack (which we apparently need for now)
```

Comment:
Let's add a concrete reason. We need it because X requires it.

Reply:
Based on the ASP.NET Community Standup from April 23rd (https://www.youtube.com/watch?v=R36j2KGBBhE) it seems that only using AddMvcCore() isn't the way to go anyway. It was an idea that never really got promoted. This will change in .NET Core 3.0, but for now this is how it's supposed to be, so I adjusted the comment accordingly.